src/styles/product-card.scss

```diff
@@ -0,0 +1,107 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__name {
+    font-size: inherit;
+    line-height: 18px;
+    height: 36px;
+    color: $main-color;
+    font-weight: 500;
+    margin-bottom: 5px;
+    width: 100%;
+  }
+
+  &__code {
+    display: flex;
+    margin: 0;
+    justify-content: flex-start;
+    width: 100%;
+    color: $secondary-color;
+    font-size: 10px;
+    margin-bottom: 18px;
+  }
+
+  &__review {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    margin-bottom: 23px;
+
+    &-text {
+      display: flex;
+      align-items: center;
+      text-align: center;
+      justify-content: center;
+      margin: 0;
+      font-size: 10px;
+      line-height: 16px;
+      font-weight: 400;
+      color: $main-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    font-size: 12px;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $secondary-color;
+      font-weight: 400;
+    }
+
+    &-price {
+      color: $main-color;
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+    background-color: $button-color;
+    color: white;
+    line-height: 16px;
+    text-transform: uppercase;
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+    transition: color 300ms, border 300ms, background-color 300ms;
+
+    &:hover {
+      border: 1px solid $button-color;
```

Comment:
No need to change the border here

You can move the border to the button - it has the same color as the normal BG color, so it would not be visible.